src/containers/App/App.js

```diff
@@ -2,14 +2,13 @@ import React, { useState } from 'react'
 import NavBar from '../../components/NavBar/NavBar.js'
 import NewPost from '../../components/NewPost/NewPost.js'
 import SearchPage from '../../components/SearchPage/SearchPage.js'
-import SplashPage from '../../components/SplashPage/SplashPage.js'
```

Comment:
Love how much cleaner this all is! Thanks for taking the time to rearrange everything and remove whatever wasn't being used/necessary